npm/vue/src/support.js

```diff
@@ -1,7 +1,4 @@
 /* eslint-env mocha */
```

Comment:
I believe this whole file is useless.

Reply:
It's used for experimentalComponent testing